src/interface.rs

```diff
@@ -272,4 +272,25 @@ impl HWIClient {
             deserialize_obj!(&output.to_string())
         })
     }
+
+    /// Install the udev rules to the local machine.
+    /// The rules will be copied from the source to the location.
+    pub fn install_udev_rules(
+        source: Option<&str>,
+        location: Option<&str>,
+    ) -> Result<HWIStatus, Error> {
```

Comment:
Instead of returning HWIStatus I'd go with returning `Result<(), Error>` (and if the status' success is false, you just return an Error), it might be a bit more complicated to code, but it's more idiomatic rust